mysql-paas-replication/azuredeploy.json

```diff
@@ -4,12 +4,21 @@
     "parameters": {
         "serverName": {
             "type": "string",
+            "defaultValue": "uniqueString",
             "metadata": {
                 "description": "Server name for the MySQL PaaS instance and it's replicas (replicas will get a '-' attached to the end with the replica number)."
             }
         },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for the MySQL PaaS components to be deployed."
+            }
+        },
         "administratorLogin": {
             "type": "string",
+            "defaultValue": "myadmin",

```

Comment:
remove the defaultValue, increases surface area

Reply:
Done